packages/cactus-test-tooling/src/main/typescript/besu/besu-test-ledger.ts

```diff
@@ -208,36 +224,16 @@ export class BesuTestLedger implements ITestLedger {
     return receipt;
   }
 
-  public async getBesuKeyPair(
-    opts: { genesisAllocIdx: number } = { genesisAllocIdx: 1 },
-  ): Promise<IKeyPair> {
-    const fn = `BesuTestLedger#getBesuKeyPair()`;
-    const genesisFile = await this.getFileContents("/opt/besu/genesis.json");
-    const { genesisAllocIdx: idx } = opts;
-    const { alloc } = JSON.parse(genesisFile);
-
-    const keyPair = Object.entries(alloc).find((_, i) => i === idx);
-
-    if (!keyPair) {
-      throw new Error(`${fn} genesis alloc missing entry with idx: ${idx}`);
-    }
-    const [pubKey, keyData] = keyPair;
-
-    const publicKey = pubKey.startsWith("0x") ? pubKey : `0x${pubKey}`;
-    if (!keyData || typeof keyData !== "object" || !("privateKey" in keyData)) {
-      throw new Error(`${fn} invalid key data within genesis alloc`);
-    }
-
-    const { privateKey } = keyData;
-    if (typeof privateKey !== "string") {
-      throw new Error(`${fn} invalid genesis alloc: privateKey must be string`);
-    }
-    const pKey0x = privateKey.startsWith("0x") ? privateKey : "0x" + privateKey;
-    return { publicKey, privateKey: pKey0x };
+  public async getBesuKeyPair(): Promise<IKeyPair> {
+    const publicKey = await this.getFileContents("/opt/besu/keys/key.pub");
+    const privateKey = await this.getFileContents("/opt/besu/keys/key");
+    return { publicKey, privateKey };
   }
 
   public async getOrionKeyPair(): Promise<IKeyPair> {
-    return this.getBesuKeyPair({ genesisAllocIdx: 2 });
+    const publicKey = await this.getFileContents("/config/orion/nodeKey.pub");
+    const privateKey = await this.getFileContents("/config/orion/nodeKey.key");
+    return { publicKey, privateKey };
```

Comment:
@LordKubaya Same question as above, is this intentional? If yes, let's talk about it.